usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -680,6 +746,10 @@ This endpoint returns a list of data that is associated with the award profile p
     `null` if the contract has no parent
 + `naics_hierarchy` (required, NAICSHierarchy, fixed-type)
 + `psc_hierarchy` (required, PSCHierarchy, fixed-type)
++ `total_account_outlay` (required, number)
++ `total_account_obligation` (required, number)
++ `account_obligations` (required, array[DEFCAmount], fixed-type)
++ `account_outlays` (required, array[DEFCAmount], fixed-type)
```

Comment:
Here and below - 
If this naming is not finalized, it appears too generic of a name for something that will be very specific to DEF codes. `account_outlays_by_defc` is good (same for obligations). Unless we want to make this an arbitrary breakdown where code could be for anything (Program Activity, object class, TAS, etc.), in which case `account_outlay_breakdown` or just `account_outlays` or `account_outlay_details` is fine.